modules/ROOT/partials/misc/autocorrect-options-notes.adoc

```diff
@@ -0,0 +1,3 @@
+NOTE: The user's preference takes precendence over the configuration at the next initialization. The user's preference is saved to `localStorage` when the user enables or disables the autocorrection via the menu item.
```

Comment:
Typo: precedence

Reply:
Do we run spell-check over this content? I think that typo should be picked up by general spell-checking.

Reply:
Typo corrected.

> Do we run spell-check over this content?

Not that I’m aware of.

Speaking only for myself, I generally run my local text editor’s in-built spell-checker of a document. Training a text editor to cleanly ignore markup is tedious, but the end result is a local spell-checker that works cleanly and quickly.